src/index.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable linebreak-style */
+
+import './style.css';
+
+const todolist = [{
+  id: '',
+  todo: 'mmmmmmmmmmmm',
+  todostatus: '',
+}];
+
+const addtodo = (inputtodo) => {
+  if (inputtodo.length < 1) {
+    const message = document.getElementById('message');
+    message.innerHTML = 'Please, type a "to do" activity';
+  } else {
+    const message = document.getElementById('message');
+    message.innerHTML = '';
+    todolist.push(
+      {
+        id: todolist.length > 0 ? todolist[todolist.length - 1].id + 1 : 1,
+        todo: inputtodo,
+        todostatus: true,
+      },
+    );
+    localStorage.setItem('todolist', JSON.stringify(todolist));
+  }
+};
+
+const removetodo = (i) => {
+  console.log(todolist[i].todo);
+};
+
+const showtodolist = (todolist) => {
+  const totallist = document.getElementById('totallist');
```

Comment:
- [ ] Please make sure that on page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the index values for each task. But currently this function is not being called on page load and the static array is not being rendered.
<img width="425" alt="Screenshot 2022-06-10 at 19 51 11" src="https://user-images.githubusercontent.com/77038610/173103847-ba56b192-532f-4217-8209-d754b67b6c47.png">